unicoremc/models.py

```diff
@@ -301,6 +309,23 @@ def get_website_manager(self):
     def frontend_url(self):
         return 'http://%s' % self.get_generic_domain()
 
+    def apollo_frontend_url(self):
+        hub = 'qa-apollo' if settings.DEPLOY_ENVIRONMENT == 'qa' else 'apollo'
+        return "%(country)s-%(app_type)s.%(hub)s.unicore.io" % {
+            'country': self.country.lower(),
+            'app_type': self.app_type,
+            'hub': hub
+        }
+
+    def content_url(self):
+        hub = 'qa-content' \
+            if settings.DEPLOY_ENVIRONMENT == 'qa' else 'content'
```

Comment:
Good idea. I'll add a setting for both `content` and `hub` as they are both hard coded. I'll leave apollo as is for now, since I plan to remove it once we're live with everything.